test/integration/etcd/etcd_cross_group_test.go

```diff
@@ -43,6 +43,7 @@ func TestCrossGroupStorage(t *testing.T) {
 		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/replicasets"] = "true"
 		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/podsecuritypolicies"] = "true"
 		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/networkpolicies"] = "true"
+		opts.APIEnablement.RuntimeConfig["extensions/v1beta1/ingresses"] = "true"
```

Comment:
this shouldn't be required, drop this change

Reply:
well, it shouldn't be required (can you verify the test passes without it?), but it's fine to put in place for the future